master/internal/cluster/authz_permissive.go

```diff
@@ -65,6 +65,23 @@ func (a *MiscAuthZPermissive) CanViewExternalJobs(
 	return (&MiscAuthZBasic{}).CanViewExternalJobs(ctx, curUser)
 }
 
+// CanModifyGlobalConfigPolicies calls the RBAC implementation and returns if
+// the user has access to modfy global task config policies.
+func (a *MiscAuthZPermissive) CanModifyGlobalConfigPolicies(
+	ctx context.Context, curUser *model.User,
+) (permErr error, err error) {
+	_, _ = (&MiscAuthZRBAC{}).CanModifyGlobalConfigPolicies(ctx, curUser)
+	return (&MiscAuthZBasic{}).CanModifyGlobalConfigPolicies(ctx, curUser)
+}
+
+// CanViewGlobalConfigPolicies calls the RBAC implementation but always allows access.
```

Comment:
This is just for my own knowledge - why do we have a function for viewing the policies if access is always allowed?

Reply:
from my understanding, if we want to change who has those permissions it will be easier in the future, or if we want to add a feature to allow the admin to configure which user has which permissions, it would be easier if we already check this permission in the API handler